product-service/src/utils/db/connect.ts

```diff
@@ -0,0 +1,23 @@
+const { Pool, Client } = require('pg');
+
+const pool = new Pool();
+
+let client;
+
+export const createConnection = async (): Promise<typeof Client> => {
+  console.log('Starting initializing db connection');
+
+  client = await pool.connect();
```

Comment:
as far as I remember we were forced to use `client` in our homework

Reply:
Yes, but I've read a bit about this topic and seems like pooling is working with Serverless approach. https://spot.io/blog/best-practices-serverless-connection-pooling-database/